site/webpack-client.config.js

```diff
@@ -1,7 +1,7 @@
 var path = require('path');
 var webpack = require('webpack');
 var MiniCssExtractPlugin = require('mini-css-extract-plugin');
-var UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
`uglifyjs-webpack-plugin` is deprecated by `terser-webpack-plugin` (see notes: https://github.com/webpack-contrib/uglifyjs-webpack-plugin#deprecated)